microservices/cmslitemetadata_to_redshift/ddl/cmslite.metadata.sql

```diff
@@ -0,0 +1,200 @@
+CREATE SCHEMA IF NOT EXISTS cmslite;
+
+-- LOOKUP TABLE FOR VALUES
```

Comment:
Can you call these Dictionary tables? That's how the README refers to them. Just keeping the language consistent to avoid confusion.

Reply:
Done